FreeRTOS_ARP.c

```diff
@@ -808,6 +808,7 @@ BaseType_t xARPWaitResolution( uint32_t ulIPAddress,
                 break;
             }
 
+            /* Decrement the count. */
             uxSendCount--;
```

Comment:
I compared your branch [AddHeaders](https://github.com/AniruddhaKanhere/FreeRTOS-Plus-TCP/tree/AddHeaders) with the HEAD version of my repo, and now it looks all good.